core/src/main/java/io/grpc/internal/ClientCallImpl.java

```diff
@@ -174,7 +171,8 @@ public void start(Listener<RespT> observer, Metadata headers) {
     }
 
     if (stream == null) {
-      stream = new DelayedStream(transportFuture, headers, listener);
+      stream = new DelayedStream(
+          this, callOptions, transportFuture, headers, listener, callExecutor, method);
```

Comment:
I was trying to keep the lock that was from ClientCallImpl.   This would preserve existing behavior no?